azure-functions/src/durable.rs

```diff
@@ -12,19 +12,24 @@ use std::{
     task::{Context, Poll, RawWaker, RawWakerVTable, Waker},
 };
 
+mod actions;
 mod creation_urls;
 mod history;
 mod management_urls;
 
+pub use actions::*;
 pub use creation_urls::*;
 pub use history::*;
 pub use management_urls::*;
 
 #[doc(hidden)]
-#[derive(Debug, Serialize, Deserialize, Default)]
+#[derive(Debug, Serialize, Default)]
 pub struct ExecutionResult {
     done: bool,
-    // TODO implements actions
+    actions: Vec<Action>,
+    output: Option<bool>,
```

Comment:
I believe the output will also be `Option<Value>`, no?

Reply:
Corresponding to this spec it's a `bool` https://github.com/Azure/azure-functions-durable-extension/wiki/Out-of-Proc-Orchestrator-Execution-Schema-Reference#output

Reply:
It's the output of the function, so it could be any serializable-to-JSON type, which bool is one.

I think this is a problem of it not being a schema, rather an example of output.

Reply:
Note in the previous example, the output was null (i.e. no output): https://github.com/Azure/azure-functions-durable-extension/wiki/Out-of-Proc-Orchestrator-Execution-Schema-Reference#sample-orchestrator-execution-result.

Reply:
Hmm...